changelog.d/9659.misc

```diff
@@ -0,0 +1 @@
+Introduce bugbear to the test suite and fix some of it's lint violations.
```

Comment:
*its*

:innocent:

Reply:
I'll fix both renditions of this. 😄

Reply:
c2d4bd62a26ea0f19445aa23367fa7ee1b4e5b08